components/src/core/components/CardTable/Cell/table-select-input.scss

```diff
@@ -2,7 +2,7 @@
   ::v-deep(.oxd-select-text) {
     border-radius: 0.5rem;
     min-height: 0;
-    padding: 0.0625rem 0.5rem;
+    padding: 4px 8px 4px;
```

Comment:
This needs to be defined in the OXD guide after discussing it with mike. Value should be taken from him considering goals as well

Reply:
Mike is ok with the changed style and asked to go with the OXD